start.R

```diff
@@ -23,6 +24,8 @@ require(stringr)
 #'
 #' Control the script's behavior by providing additional arguments:
 #'
+#' --help, -h: show this text and exit
+#'
 #' --debug, -d: start a debug run with cm_nash_mode = debug
 #'
 #' --interactive, -i: interactively select run(s) to be started
```

Comment:
the new `-q` option seems to be missing here in the help text. And I would rename it to simply `quick`. What else should it start if not a `run`?